test/server.test.js

```diff
@@ -2755,9 +2743,7 @@ test('should increment/decrement inflight request count', function(t) {
     });
 });
 
-test('should increment/decrement inflight request count for concurrent reqs', function(
-    t
-) {
+test('should increment/decrement inflight request count for concurrent reqs', function(t) {
```

Comment:
Pretty sure this exceeds 80 columns?

Reply:
Good catch, something went wrong with this `prettier` release. I'm going to dig deeper.